RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -0,0 +1,427 @@
+using Microsoft.AspNetCore.Http;
+using Neo.IO.Caching;
+using Neo.IO.Data.LevelDB;
+using Neo.IO.Json;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence.LevelDB;
+using Neo.Wallets;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Neo.Ledger;
+using Neo.Persistence;
+using Snapshot = Neo.Persistence.Snapshot;
+
+namespace Neo.Plugins
+{
+    public class RpcSystemAssetTrackerPlugin : Plugin, IPersistencePlugin, IRpcPlugin
+    {
+        private const byte SystemAssetUnspentCoinsPrefix = 0xfb;
+        private const byte SystemAssetSpentUnclaimedCoinsPrefix = 0xfc;
```

Comment:
We should have a global prefix for plugins, and concat this prefix with this one, in order to prevent collisions.

Reply:
Well it can’t use the same database as the core is using, so it won’t collide with it. The method it uses takes a 1 byte prefix. We could have an additional prefix, but it doesn’t really seem needed right now.

Reply:
If is a different storage there is no problem.